src/config/network/dhcpcd.md

```diff
@@ -7,6 +7,12 @@ can enable the `dhcpcd` service.
 # ln -s /etc/sv/dhcpcd /var/service
 ```
 
+And then start it with [sv(8)](https://man.voidlinux.org/sv.8):
```

Comment:
`sv start` only works if `runsv` for the directory is already running, which means the service is already running or there is a `down` file in the service directory.
So this is redundant, you don't need to start a service after activating it.

Reply:
Oops, I'd been doing that this whole time haha. Thanks